GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
```

Comment:
Command - executable; directive that performs a certain task. https://github.com/operable/cog/wiki/Commands-and-Bundles#commands